include/lib/output.inc.php

```diff
@@ -56,9 +56,9 @@ function _AC() {
 		// replaced the preg_match with a test of the substring.
 		$sub_arg = substr($args[0], 0, 7); // 7 is the shortest type of msg (AC_INFO)
 		if (in_array($sub_arg, array('AC_ERRO','AC_INFO','AC_WARN','AC_FEED','AC_CONF'))) {
-			global $_base_path, $addslashes;
+			global $_base_path;
 
-			$args[0] = $addslashes($args[0]);
+			$args[0] = addslashes($args[0]);
```

Comment:
Unnecessary to add `addslashes` as it will be handled in `languageTextDAO` 3 lines down. Also, calling `addslashes` directly is problematic without checking `magic_quotes_gpc `. See [PHP manual](http://php.net/manual/en/function.addslashes.php).